.github/workflows/tests.yml

```diff
@@ -3,7 +3,7 @@ name: Tests
 on: [push, pull_request]
 
 jobs:
-  changeset:
+  changeset-lint:
```

Comment:
I'd be ok just calling this lit and considering checking for a changeset a lint step. @kevinpschaaf ?

Reply:
s/lit/lint
yeah

Reply:
so just `lint` ? confirming

Reply:
done!